pkg/operator/ceph/object/objectstore.go

```diff
@@ -875,11 +875,9 @@ func getDashboardUser(context *Context) (ObjectUser, error) {
 		SystemUser:  true,
 	}
 
-	if !context.CephClusterSpec.External.Enable {
```

Comment:
Maybe these checks were added so we can prevent reconciling External-cluster against rgw reconciliation process as it doesn't work probably for now.

Reply:
I think this is a good question to investigate. This is related to dashboard. Do we know if dashboard will work for external clusters?

Reply:
getDashboardUser() is called from enableDashboard() which is called from createOrUpdateStore(). The createOrUpdateStore() called only for internal cluster, so we are not enabling dashboard for external rgw servers